src/styles/header.css

```diff
@@ -0,0 +1,70 @@
+/* Styles go here */
+
+body {
+  --hover-color: #00acdc;
+}
+
+.logo {
```

Comment:
The '.logo' selector should follow BEM naming convention. Change the selector to '.header__logo'.